lib/winrm-fs/core/file_transporter.rb

```diff
@@ -443,6 +443,16 @@ def stream_upload(input_io, dest)
 
         def stream_command(encoded_bytes)
           <<-EOS
+            # Powershell caches ScritBlocks in a dictionary
```

Comment:
Probably better to move the comment outside the powershell script to avoid uploading it?